omc3/optics_measurements/phase.py

```diff
@@ -66,6 +66,14 @@ def calculate(
                                                                              no_errors)
         dfs = free_dfs + drv_dfs
 
+    if phase_advances["MEAS"].count() < 3:
+        LOGGER.warning("Dear user, phase analysis yields less than 3 BPMs.")
```

Comment:
I'd write it into one single log with `\n` to separate lines. That makes it more obvious that it's one message. Also, `Dear user` might be too much